src/components/RegistrationTable/EditCell.tsx

```diff
@@ -32,18 +57,53 @@ export const EditCell: React.FC<EditCellProps> = ({ row }) => {
+    ;
+    // The 'test' user from the database was used here for the responses
+    const mockResponses: UserQuestionIDs = {
+        "ee2b5b93-3792-4332-ba83-24b995f12094": { S: "asdf" },
+        "04f58eea-1861-4da3-8e4c-8d8e994ce8ba": { S: "asdf" },
+        "bede9713-17cf-4bb9-b362-8c30a1e5b543": { S: "UBC" },
+        "cb1a1e83-f581-473c-97e0-48e9a88d7d20": { S: "I would not like to participate" },
+        "ea4a8fa7-7bcf-4f86-af60-27ac61c7680b": { S: "https://drive.google.com/file/d/1Q60JBkifak4gSrZL65KvafCPVA2HwcZ4/view?usp=drivesdk" },
+        "013bb98c-4286-4649-bbb9-fbc27185925c": { S: "asdf" },
+        "24f0b385-d92e-43be-92a3-2b225964a778": { S: "No" },
+        "a04e1064-65b2-4873-bede-9c4d5cbd32c3": { S: "No" },
+        "49512eca-eec6-4639-b70c-f60f2e0a9371": { S: "No" },
+        "f33ba987-6a5f-4ea2-9ba9-ca5f087e6fef": { S: "No" },
+        "f83944a8-97e2-421c-903c-aac91106fe34": { S: "I didn't submit a project" }
+    };
```

Comment:
I don't think we work with DynamoDB-JSON marshalled data in our frontend, it's all dealt with in the backend. Can you confirm @AllanT102 @bennypc ?

Reply:
Yeah we don't. This is a mock response sent by the backend

<img width="392" alt="Screenshot 2024-08-12 at 1 16 24 AM" src="https://github.com/user-attachments/assets/0413e037-fc49-44e6-b437-873ea82250f9">

Reply:
ahh okay i see i'll fix that asap thanks guys 🙏